core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -540,4 +540,14 @@ describe('Contributor dashboard Admin page', () => {
     expect(modalSpy).toHaveBeenCalledWith(UsernameInputModal);
     expect(openRoleEditorSpy).toHaveBeenCalledWith('user1');
   }));
+
+  it('should rearrange language name', () => {
```

Comment:
What's the result of rearranging the language name? That should be explicitly stated in the test name.

Something like `'should move language name in English to the front'` or `'should start any language by its English name'` explicitly states the end goal of rearranging the language name.

Reply:
Done.